### ISSUE TITLE
Parser does not read the types of class attributes

### ISSUE THREAD

**Issue by XsongyangX** [Enteee/plantuml-parser#78]
I am using this tool to parse class diagrams. The tool doesn't parse the types of attributes.

NodeJS script:
```js
import { parse, parseFile, formatters } from 'plantuml-parser';

// Example PlantUML
const data = `
@startuml
!theme plain
class Publication {
  title : String
}
@enduml
`;

// parse PlantUML
const result = parse(data);

// Format and print parse result
console.log(
  formatters.default(result)
);
```

Output:
```
[
  {
    "elements": [
      {
        "name": "Publication",
        "title": "Publication",
        "isAbstract": false,
        "members": [
          {
            "name": "title",
            "isStatic": false,
            "accessor": "+",
            "type": ""
          }
        ],
        "extends_": [],
        "implements_": [],
        "generics": [],
        "stereotypes": []
      }
    ]
  }
]
```

The `String` type of the `title` attribute is not present in the output.

**Comment by Enteee** [comment]
good catch :+1:  will try to get this fixed

**Comment by Enteee** [comment]
thank you for reporting this. This was implemented with #80 .
